#
# Guidance Status
#
uint8 guidance_mode
float32[] poly
float32 x_min
float32 x_max             
float32 lookahead_dist
float32 lookahead_time
float32 carrot_x
float32 carrot_y
float32 R
float32 lin_sp
float32 ang_sp